spec/features/smart_answer/calculate_your_holiday_entitlement_flow_spec.rb

```diff
@@ -0,0 +1,382 @@
+    click_button "Continue"
+  end
+
+  def what_is_your_starting_date(day, month, year)
+    expect(page).to have_selector("h1", text: headings[:what_is_your_starting_date])
+    fill_in "response[day]", with: day
+    fill_in "response[month]", with: month
+    fill_in "response[year]", with: year
+    click_button "Continue"
+  end
+
+  def what_is_your_leaving_date(day, month, year)
+    expect(page).to have_selector("h1", text: headings[:what_is_your_leaving_date])
+    fill_in "response[day]", with: day
+    fill_in "response[month]", with: month
+    fill_in "response[year]", with: year
+    click_button "Continue"
+  end
+
+  def when_does_your_leave_year_start(day, month, year)
+    expect(page).to have_selector("h1", text: headings[:when_does_your_leave_year_start])
+    fill_in "response[day]", with: day
+    fill_in "response[month]", with: month
+    fill_in "response[year]", with: year
+    click_button "Continue"
+  end
+
+  def how_many_hours_per_week(response)
+    expect(page).to have_selector("h1", text: headings[:how_many_hours_per_week])
+    fill_in "response", with: response
+    click_button "Continue"
+  end
+
+  def how_many_days_per_week_for_hours(response)
+    expect(page).to have_selector("h1", text: headings[:how_many_days_per_week_for_hours])
+    fill_in "response", with: response
+    click_button "Continue"
+  end
+
+  def shift_worker_basis(response)
+    expect(page).to have_selector("h1", text: headings[:shift_worker_basis])
+    choose response
+    click_button "Continue"
+  end
+
+  def shift_worker_hours_per_shift(response)
+    expect(page).to have_selector("h1", text: headings[:shift_worker_hours_per_shift])
+    fill_in "response", with: response
+    click_button "Continue"
+  end
+
+  def shift_worker_shifts_per_shift_pattern(response)
+    expect(page).to have_selector("h1", text: headings[:shift_worker_shifts_per_shift_pattern])
+    fill_in "response", with: response
+    click_button "Continue"
+  end
+
+  def shift_worker_days_per_shift_pattern(response)
+    expect(page).to have_selector("h1", text: headings[:shift_worker_days_per_shift_pattern])
+    fill_in "response", with: response
+    click_button "Continue"
+  end
+
+  def statutory_entitlement(count, basis = "days")
+    "The statutory holiday entitlement is #{count} #{basis} holiday."
+  end
+
+  def statutory_entitlement_shifts(shifts, hours)
+    "The statutory holiday entitlement is #{shifts} shifts for the year. Each shift being #{hours} hours."
+  end
+
+  def statutory_entitlement_hours(hours)
+    "The statutory entitlement is #{hours} hours holiday."
+  end
+
+  def statutory_entitlement_compressed(hours, minutes)
+    "The statutory holiday entitlement is #{hours} hours and #{minutes} minutes holiday for the year. Rather than taking a day’s holiday it’s 8 hours and 0 minutes holiday for each day otherwise worked."
+  end
+
+  before do
+    # 2020 was a leap year
+    Timecop.freeze(Time.zone.local(2020))
```

Comment:
See comment below - I don't think we need to be freezing time anymore

Reply:
Yes, absolutely right - excellent point.  Removed `Timecop`